File: Programming-Basics-With-JavaScript/Homeworks-And-Labs/03.Conditional-Statements-Advanced/02-exercise/05.journey.js
function journey(input) {

    const budget = Number(input[0]);
    const season = input[1];

    let destination = 0;
    let place = 0;
    let sum = 0;

    if (budget <= 100) {
        destination = "Bulgaria";
        switch (season) {
            case "summer":
                sum = budget * 0.30;
                place = "Camp";
                break;
            case "winter":
                sum = budget * 0.70;
                place = "Hotel";
                break;
        }
    } else if (budget <= 1000) {
        destination = "Balkans";
        switch (season) {
            case "summer":
                sum = budget * 0.40;
                place = "Camp";
                break;
            case "winter":
                sum = budget * 0.80;
                place = "Hotel";
                break;
        }
    } else if (budget > 1000) {
        destination = "Europe";
        sum = budget * 0.90;
        place = "Hotel";
    }

    console.log(`Somewhere in ${destination}`);
    console.log(`${place} - ${sum.toFixed(2)}`);
}
journey(["1500", "summer"]);

File: Python_Basics/Homeworks-And-Labs/03.conditional_statements_advanced/02.exercise/05.journey.py
budget = float(input())
season = input()

destination = 0
place = 0
sum = 0

if budget <= 100:
    destination = "Bulgaria"
    if season == 'summer':
        sum = budget * 0.30
        place = "Camp"
    elif season == 'winter':
        sum = budget * 0.70
        place = "Hotel"
elif budget <= 1000:
    destination = "Balkans"
    if season == 'summer':
        sum = budget * 0.40
        place = "Camp"
    elif season == 'winter':
        sum = budget * 0.80
        place = "Hotel"
elif budget > 1000:
    destination = "Europe"
    sum = budget * 0.90
    place = "Hotel"

print(f'Somewhere in {destination}')
print(f'{place} - {sum:.2f}')
